docs/guides/index.md

```diff
@@ -266,6 +266,9 @@ image, or a 360&deg; video. For example, a dark gray background would be:
 
 ## Applying an Image Texture
 
+> From this point on, you must be serving your HTML over a Web server (such as a local
```

Comment:
I'd just put something like `Make sure you're [serving your HTML using a local server](../introduction/getting-started.md#using-a-local-server) for textures to load properly.`